### PR TITLE
Frontend restructured and timer fix

### PR DESCRIPTION
**Pull request by lember103**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,19 +1,9 @@
...
 
-    const [minutes, setMinutes] = useState("")
-
-    const handleClick = () => {
-        postMinutes(minutes).then(response => console.log(response))
-    }
-
     return (
-        <form>
-            <input onChange={event => setMinutes(event.target.value)} placeholder={"Minuten"}/>
```
Comment (Haferkorn) [inline frontend/src/App.js:14]: I think it is best-practice to always add the input type to the html element and a label

File: frontend/src/components/PumpAction.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    const [minutes, setMinutes] = useState("")
+
+    const updateTimer = () => {
+        postMinutes(minutes).then(response => console.log(response))
+    }
+
+    return(
+        <Styled>
+            <input onChange={event => setMinutes(event.target.value)} placeholder={"Minuten"}/>
```
Comment (Haferkorn) [inline frontend/src/components/PumpAction.js:15]: as above

**Review by Haferkorn — APPROVED** [review]
Nice work :-)

File: backend/src/main/java/de/neuefische/backend/model/PumpTimer.java [hunk 3]
```diff
@@ -3,12 +3,10 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.time.Duration;
-
 @Data
 @NoArgsConstructor
```
[the remark this answers is not in the document]
Comment (lember103) [reply backend/src/main/java/de/neuefische/backend/model/PumpTimer.java:7]: Not at the the moment but I keep it in mind

File: frontend/src/App.js (hunk 1 above)
Comment (lember103) [reply frontend/src/App.js:14]: Thanks for the advice :)

**Pull request merged**